lnschema_core/_core.py

```diff
@@ -497,10 +497,12 @@ def replace(self, data: Union[Path, str, pd.DataFrame, ad.AnnData], source: Opti
         self.hash = kwargs["hash"]
         self.suffix = kwargs["suffix"]
         self.source = kwargs["source"]
+        self.key = kwargs["key"]
+
         self._local_filepath = privates["local_filepath"]
         self._cloud_filepath = privates["cloud_filepath"]
         self._memory_rep = privates["memory_rep"]
-        self._to_store = True
+        self._to_store = not privates["check_path_in_storage"]
```

Comment:
If this is not set to True, it won't overwrite/upload. So, in most cases now, `.replace()` will actually not do what we want it to do.

Are you sure you'd like to set it to `not privates["check_path_in_storage"]`? 🤔

Reply:
Right, now I understand! This is a legit user story. 🤔